.circleci/config.yml

```diff
@@ -51,29 +51,18 @@ commands:
             $CMD
 
   run-grakn-server:
+    parameters:
+      rbe:
+        description: use RBE while building
+        type: boolean
+        default: false
     steps:
-      - run: bazel build @graknlabs_grakn//:distribution
-      - run: unzip bazel-genfiles/external/graknlabs_grakn/grakn-core-all.zip -d bazel-genfiles/dist/
+      - run: bazel build @graknlabs_grakn_core//:distribution <<# rbe >> --config=rbe <</ rbe >>
```

Comment:
have you considered using the `bazel` command which is defined in the `command` section at around line 35 of this file?

it can be used like this:
```
      - bazel:
          command: bazel build @graknlabs_grakn_core//:distribution
```

Reply:
@lolski not really :) My command lets you specifically turn RBE on/off whereas `bazel` CircleCI command turns on RBE if it exists. I'll remove that command definition.